packages/third-party/tailwind-preset/README.md

```diff
@@ -0,0 +1,54 @@
+# Lynx Tailwind Preset
+
+A Tailwind CSS preset specifically designed for Lynx, ensuring that only CSS properties supported by Lynx are available as Tailwind utilities.
```

Comment:
```suggestion
A Tailwind CSS V3 preset specifically designed for Lynx, ensuring that only CSS properties supported by Lynx are available as Tailwind utilities.
```

Reply:
will do this later. I'll append another PR moving this into `packages` so this PR is easier to review